sim/rogue/poisons.go

```diff
@@ -67,6 +67,16 @@ func (rogue *Rogue) improvedPoisonsBonusProcChance() float64 {
 	return 0.02 * float64(rogue.Talents.ImprovedPoisons)
 }
 
+// p8 DPS tier bonus helper function
+func trackTotalUniquePoisons(aura *core.Aura, rogue *Rogue) {
+	aura.ApplyOnGain(func(aura *core.Aura, sim *core.Simulation) {
+		rogue.PoisonsActive[aura.Unit.UnitIndex]++
+	})
+	aura.ApplyOnExpire(func(aura *core.Aura, sim *core.Simulation) {
```

Comment:
🔨 Tiny knit. We can chain these calls because they return the Aura pointer
```suggestion
	}).ApplyOnExpire(func(aura *core.Aura, sim *core.Simulation) {
```